LM75B.lib

```diff
@@ -1 +0,0 @@
-https://developer.mbed.org/users/neilt6/code/LM75B/#7ac462ba84ac
```

Comment:
It is being replaced by the more generic I2CEeprom.lib file. This new library also future proofs us and enables use of the File System API's.